.env.example

```diff
@@ -8,6 +8,27 @@ OPENSEARCH_USER=admin
 OPENSEARCH_PASSWORD=admin
 OPENSEARCH_URL=http://opensearch-node1:9200
 OPENSEARCH_INDEX=navigator
+OPENSEARCH_REQUEST_TIMEOUT=30
+OPENSEARCH_PREFERENCE=prototype_user  # TODO: document what this means
+OPENSEARCH_USE_SSL=False
+OPENSEARCH_VERIFY_CERTS=False
+OPENSEARCH_SSL_WARNINGS=False
+
+# Opensearch query/index settings
+OPENSEARCH_INDEX_INNER_PRODUCT_THRESHOLD=70.0
+OPENSEARCH_INDEX_MAX_DOC_COUNT=100
+OPENSEARCH_INDEX_MAX_PASSAGES_PER_DOC=10
+OPENSEARCH_INDEX_KNN_K_VALUE=10000
+OPENSEARCH_INDEX_N_PASSAGES_TO_SAMPLE_PER_SHARD=5000
+OPENSEARCH_INDEX_NAME_BOOST=100
+OPENSEARCH_INDEX_DESCRIPTION_BOOST=10
+OPENSEARCH_INDEX_NAME_KEY=for_search_action_name
+OPENSEARCH_INDEX_DESCRIPTION_KEY=for_search_action_description
+OPENSEARCH_INDEX_DESCRIPTION_EMBEDDING_KEY=action_description_embedding
+OPENSEARCH_INDEX_INDEX_KEY=action_name_and_id
+OPENSEARCH_INDEX_TEXT_BLOCK_KEY=text_block_id
+OPENSEARCH_INDEX_ENCODER=msmarco-distilbert-dot-v5
+OPENSEARCH_INDEX_EMBEDDING_DIM=768
```

Comment:
this can be deleted ☺️

Reply:
I added that to the env file because it was really the only part of the index config I was aware of that wasn't configurable. it's not used by any of the backend code, but I was hoping this might be the start of shared config between the `search-index` code and the `backend` ?